src/pages/@owner/estimate/hooks/use-time.ts

```
@@ -0,0 +1,179 @@
+import type { EstimateFormData } from '@pages/@owner/estimate/utils/estimate.schema';
+import { isDateOverlapping } from '@pages/@owner/food-truck-form/utils/is-date-over-lapping';
+import {
+  ESTIMATE_ERROR_MESSAGE,
+  ESTIMATE_MAX_LENGTH,
+} from '@pages/@owner/estimate/constants/estimate';
+
+//time, date 관련 로직
+interface UseEstimateTimeProps {
+  formData: EstimateFormData;
+  setValue: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)